cypress/integration/movies.test.js

```diff
@@ -24,7 +24,7 @@ describe("has a functioning Movies page", () => {
   
     it("selects a movie and displays title and summary", () => {
       cy.get("select").select("Ponyo")
-  
+      cy.wait(1000)
```

Comment:
I know I told you to put this here but next time don't push any changes in the test file